class.jetpack.php

```diff
@@ -5116,15 +5114,7 @@ public static function register() {
 		 */
 		do_action( 'jetpack_site_registered', $registration_details->jetpack_id, $registration_details->jetpack_secret, $jetpack_public );
 
-		// Initialize Jump Start for the first and only time.
-		if ( ! Jetpack_Options::get_option( 'jumpstart' ) ) {
-			Jetpack_Options::update_option( 'jumpstart', 'new_connection' );
-
-			$jetpack = Jetpack::init();
```

Comment:
Need to investigate what was going on here and if `jumpstart` option is used to somehow detect the state of Jetpack, not just state of Jumpstart?